ibverbs/src/lib.rs

```diff
@@ -1488,26 +1530,16 @@ impl<'res> QueuePair<'res> {
     ///
     /// [1]: http://www.rdmamojo.com/2013/01/26/ibv_post_send/
     #[inline]
-    pub unsafe fn post_send<T, R>(
+    pub unsafe fn post_send<'a>(
         &mut self,
-        mr: &mut MemoryRegion<T>,
-        range: R,
+        local: &[LocalMemorySlice<'a>],
```

Comment:
I think this needs to be `&mut []` so that callers can't accidentally use the same local memory region to multiple calls at once.